### ISSUE TITLE
Where are the loss functions defined?

### ISSUE THREAD

**Issue by saravanabalagi** [JunjH/Revisiting_Single_Depth_Estimation#3]
In the paper, three loss functions, l_depth, l_grad, l_normal are described. But I'm having trouble finding those out in the code.

**Comment by saravanabalagi** [comment]
Also, what are the values used for lambda and mu in the experiment?

![image](https://user-images.githubusercontent.com/8567893/43681229-d77d2478-9845-11e8-9bc5-81cdab91f560.png)

**Comment by hypercost** [comment]
There is only evaluate code in this repo.
You can find training loss definition in https://github.com/xanderchf/MonoDepth-FPN-PyTorch

**Comment by saravanabalagi** [comment]
Thanks a bunch. 

Is [that](https://github.com/xanderchf/MonoDepth-FPN-PyTorch) the original implementation of this paper though? I do not see the original authors there, and I also get to see that this paper was referenced from a third person's perspective as Hu et al.

**Comment by JunjH** [comment]
Code for training and test is available now.
